tcs-service/src/test/java/com/transformuk/hee/tis/tcs/service/service/impl/PostServiceImplTest.java

```diff
@@ -443,6 +440,21 @@ void deleteShouldDeleteOneInstanceById() {
     verify(postRepositoryMock).deleteById(1L);
   }
 
+  @Test
+  void deleteShouldFailWhenPostIsReconciledWithESR() {
```

Comment:
Please confirm this is true.
Please also confirm the expected behaviour for the reconciliation events (should deletes cascade), if a delete should succeed when post is reconciled with ESR.

Reply:
Yes it does